### DISCUSSION TITLE
Setting Different Storage Type for OfficeRuntime.Storage

### DISCUSSION THREAD

**Discussion by AdvaSoftware** [OfficeDev/office-js#2976] [Q&A]
Hi. I was wondering if its possible to make OfficeRuntime.Storage use IndexedDB or other storage types other than localstorage. My solution needs to transfer a large amount of data from custom functions to the main app context and localstorage doesn't seem to handle the load that well (I see that its promises break).

**Comment by demyte** [comment]
@AdvaSoftware If you used a [shared runtime addin](https://learn.microsoft.com/en-us/office/dev/add-ins/develop/configure-your-add-in-to-use-a-shared-runtime)  you should be able to use IndexedDB or a wrapper around it like [https://dexie.org/](Dexie)
